Sources/Attribution/AttributionPoster.swift

```diff
@@ -130,6 +123,23 @@ class AttributionPoster {
         }
     }
 
+    // should match OS availability in https://developer.apple.com/documentation/ad_services
+    @available(iOS 14.3, macOS 11.1, macCatalyst 14.3, *)
+    @available(tvOS, unavailable)
+    @available(watchOS, unavailable)
+    func postAdServicesTokenIfNeeded() {
+        let latestTokenSent = latestNetworkIdAndAdvertisingIdentifierSent(network: .adServices)
+        guard latestTokenSent == nil else {
+            return
+        }
```

Comment:
Since `latestTokenSent` isn't used in the method we don't need a variable for it (it's kind of confusing because it's always going to be `nil`. I'd change this to:
```suggestion
        guard self.latestNetworkIdAndAdvertisingIdentifierSent(network: .adServices) == nil else {
            return
        }
```

Reply:
Bump